c_src/driverkit.hpp

```diff
@@ -67,7 +68,11 @@ void block_till_listener_init();
 void close_registered_devices();
 void notify_start_loop();
 int  init_sink();
-void exit_sink();
+#ifdef USE_KEXT
+    int exit_sink();
+#else
+    void exit_sink();
```

Comment:
How about we change the `void exit_sink()` to `int` too and have it just return 0.
I think I'd like it better to have a singe exit_sink function declared.